### ISSUE TITLE
inquiretest always throws Exception : cannot find service 

### ISSUE THREAD

**Issue by ShiverZm** [Agamnentzar/bluetooth-serial-port#16]

**Comment by Agamnentzar** [comment]
Do you have bluetooth receiver connected to your computer ?

**Comment by ShiverZm** [comment]
the same,
always throws Exception:
+ Message "(10108) This service does not exist. The service is not found in the specified namespace. \r\n" std::basic_string,std::allocator >

**Comment by ShiverZm** [comment]
in a addition, i comment this "cout << "\tchannel ID: " << inq->SdpSearch(d.address) << endl",it can work

**Comment by ShiverZm** [comment]
And win10-connect-Android  always throws Exception：Cannot Connect: (10064) The socket operation failed because the target host was down.   it broke down at "connect" return -1, i surely  they are connected.

My iphone never is like this, it can work  but that is not i want. i want comunicate with android app.

**Comment by Agamnentzar** [comment]
Yeah, SdpSearch seems to not work correctly, you can just use value of `1` for channel id. Also be aware that this library connect using COM protocol, so it will only connect to devices that support that protocol.

**Comment by ShiverZm** [comment]
Could you mind telling me  something about bluetooth-add-service by sdp?i dont know how to nearly

**Comment by Agamnentzar** [comment]
I don't know anything about that
